### DISCUSSION TITLE
Features // How to arrange objects in layers

### DISCUSSION THREAD

**Discussion by LeHarry** [frappe/print_designer#169] [Q&A] [answered]
![grafik](https://github.com/frappe/print_designer/assets/160015230/5291ab5f-ea5f-4340-91f7-232084bc726f)

Q: How do you arrange the order of objects / how do you layer within Printdesigner to correct overlap?

**Comment by maharshivpatel** [comment, marked as answer]
@LeHarry this is not implemented yet. i will add it soon.
